src/common/InlineDateTime.tsx

```diff
@@ -0,0 +1,18 @@
+import { format } from "../logion-chain/datetime";
+
+export interface Props {
+    dateTime?: string;
+}
+
+export default function InlineDateTime(props: Props) {
+    let voidedOn;
```

Comment:
I would name this var differently, this component could be used in a different context.
```suggestion
    let formattedDateTime;
```

Reply:
Indeed, my bad.